postgres/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM postgres:16-alpine
+ENV POSTGRES_PASSWORD=petclinic
+ENV POSTGRES_DB=petclinic
```

Comment:
Let's change approach here. Add `POSTGRES_PASSWORD` and ` POSTGRES_DB` as `ARG` for this Dockerfile. This way, we can provide these 2 variables during `docker build`. This gives us more flexibillity while building image to change credentials.

While not mandatory in this case, but it is nice chance to try using `--build-arg`
* https://docs.docker.com/build/guide/build-args/